types/html-escaper/index.d.cts

```diff
@@ -0,0 +1,7 @@
+interface htmlEscaperStatic {
```

Comment:
I am curious if you can avoid this by importing from the index file instead of duplicating.

Reply:
Duplication is fixed by importing `.d.cts` typedef into `.d.ts`.